comparative-politics.csl

```diff
@@ -0,0 +1,164 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>Comparative Politics</title>
+    <id>http://www.zotero.org/styles/comparative-politics</id>
+    <link href="http://www.zotero.org/styles/comparative-politics" rel="self"/>
+    <link href="http://www.zotero.org/styles/cultural-geographies" rel="template"/>
+    <link href="http://jcp.gc.cuny.edu/submission/style-sheet/" rel="documentation"/>
+    <author>
+      <name>Patrick O'Brien</name>
+      <email>obrienpat86@gmail.com</email>
+    </author>
+    <category citation-format="note"/>
+    <category field="political_science"/>
+    <category field="social_science"/>
+    <issn>0010-4159</issn>
+    <updated>2017-07-27T14:08:52+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="author">
+    <names variable="author">
+      <name and="text" delimiter-precedes-last="never" sort-separator=" "/>
+      <label form="short" prefix=" (" suffix=")"/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="editor">
+    <names variable="editor">
+      <name and="text" delimiter-precedes-last="never" initialize-with="."/>
```

Comment:
I'm seeing Stanley Rosen and Peter Hays Gries, eds. in the instructions, i.e. without `initialize-with`